scss/tech.scss

```diff
@@ -0,0 +1,34 @@
+
+.border{
+width: 200px;
+height: 300px;
+border: solid 1px;
+border-radius: 10px;
+border-color: #bed23c;
+}
+
+.tech{
+background-color: #bed23c;
+border-radius: 10px;
+display: flex;
+justify-content: space-around;
+align-items: center;
+flex-direction: column;
+width: 200px;
+height: 300px;
+padding: 10px;
+position: relative;
+left: 10px;
+top: 10px;
+}
+.tech img {
```

Comment:
Add a new line before this one (24).
I do not recommend using html elements in styles, do not use `img` for example